src/Native/pkg/runtime.native.System.IO.Compression/win7/runtime.native.System.IO.Compression.pkgproj

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+    <PropertyGroup>
+        <Version>4.0.1</Version>
+        <PackageTargetRuntime>win7-$(PackagePlatform)</PackageTargetRuntime>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <File Include="$(WinNativePath)ProjectK\Runtime\clrcompression.dll">
```

Comment:
You should remove "ProjectK\Runtime" from this now. Please do verify that in TFS builds.